services/workflows-service/src/workflow/workflow.controller.internal.test.ts

```diff
@@ -0,0 +1,160 @@
+import { BaseFakeRepository } from '../../../../test-utils/src/base-fake-repository';
+
+import { WorkflowControllerInternal } from './workflow.controller.internal';
+import { WorkflowService } from './workflow.service';
+import { WorkflowDefinitionModel } from './workflow-definition.model';
+import { WorkflowEventEmitterService } from './workflow-event-emitter.service';
+
+class FakeWorkflowRuntimeDataRepo extends BaseFakeRepository {
+  constructor() {
+    super(Object);
+  }
+}
+
+class FakeWorkflowDefinitionRepo extends BaseFakeRepository {
+  constructor() {
+    super(WorkflowDefinitionModel);
+  }
+}
+
+function buildWorkflowDeifintion(sequenceNum) {
+  return {
+    id: sequenceNum.toString(),
+    name: `name ${sequenceNum}`,
+    version: sequenceNum,
+    definition: {
+      initial: 'initial',
+      states: {
+        initial: {
+          on: {
+            COMPLETE: 'completed',
+          },
+        },
+        completed: {
+          type: 'final',
+        },
+      },
+    },
+    definitionType: `definitionType ${sequenceNum}`,
+    createdAt: new Date(),
+    updatedAt: new Date(),
+  };
+}
+
+describe('WorkflowControllerInternal', () => {
+  let controller;
+  let workflowRuntimeDataRepo;
+  let eventEmitterSpy;
+  const _ = Symbol('not in use');
```

Comment:
Change to fakeUser or mockUser, `not in use` should also be renamed.

Reply:
changed to numbUserInfo